src/reason_pprint_ast.ml

```diff
@@ -857,17 +857,18 @@ let default = new Pprintast.printer ()
 type funcReturnStyle =
   | ReturnValOnSameLine
 
-let rec detectJSXComponent e attributes l =
-  match (e, attributes) with
+let rec detectJSXComponent e attributes l = None
```

Comment:
What's happening here? Simply having this change (without the rest) doesn't change break any unit test?